### PR TITLE
returning a blank string if the Enum Label resolves to undefined

### PR DESCRIPTION
**Pull request by ChristopherCarson**
resolves https://github.com/autoinvent/conveyor/issues/69

### CODE REVIEW

File: src/delete/DeleteDetail.js [hunk 1]
```diff
@@ -30,7 +30,8 @@ const getRowFields = (schema, modelName, node, nodeOrder) => {
 
     if (exclusionCondition(key)) {
       if (schema.isEnum(modelName, key)) {
-        return schema.getEnumLabel(modelName, key, value)
+        const label = schema.getEnumLabel(modelName, key, value)
+        return label === undefined ? '' : label
```
Comment (myurch) [inline src/delete/DeleteDetail.js:34]: what if 'label' is null?

**Comment by myurch** [comment]
could this DeleteReview issue possibly occur with other field types? 

perhaps there's a way to fix this at the higher level. is there a reasonable way that, if a value for a column is 'undefined' it could simply resolve as an empty column in the table?

File: src/delete/DeleteDetail.js (hunk 1 above)
Comment (ChristopherCarson) [reply src/delete/DeleteDetail.js:34]: null is OK. It won't be stripped out, miss-aligning the columns. A little further in the same file only undefined values are removed.
R.reject(val => val === undefined)

**Comment by ChristopherCarson** [comment]
> could this DeleteReview issue possibly occur with other field types?
> 
> perhaps there's a way to fix this at the higher level. is there a reasonable way that, if a value for a column is 'undefined' it could simply resolve as an empty column in the table?

Nothing is coming back from the query undefined. Columns and rows with no values begin as blank. The logic in getRowFields() creates undefined values in columns if a header has no matching data in the query node. That is why those are removed... The headers then matches the columns. We can skip the schema.isEnum() check if a column matches a header and is blank. That would keep it blank all the way to being displayed.

**Pull request merged**